packages/backend-common/src/reading/FetchUrlReader.ts

```diff
@@ -60,11 +69,29 @@ export class FetchUrlReader implements UrlReader {
               }
             : (_url: URL) => true;
           const host = allowConfig.getString('host');
-          if (host.startsWith('*.')) {
-            const suffix = host.slice(1);
-            return (url: URL) => url.host.endsWith(suffix) && checkPath(url);
+          const [hostname, port] = host.split(':');
+
+          let checkPort = (url: URL) => !url.port;
+          if (port) {
+            const isRange = port.includes('-');
+            const range = port.split('-').map(v => parseInt(v, 10)) as [
```

Comment:
Could we make this a little bit more strict? thinking validate that we're actually getting numbers back, throwing an error if not. I'm thinking you could wrap this logic in a `parsePortPredicate` function or something like that that returns the appropriate lambda function.